src/app.rs

```diff
@@ -25,15 +25,12 @@ pub enum AppModes {
     HelpPage,
 }
 
-pub fn get_frame_lines(
-    tf_listener: Arc<rustros_tf::TfListener>,
-    static_frame: String,
-    axis_length: f64,
-) -> Vec<Line> {
-    let tf = tf_listener
-        .lookup_transform(&static_frame, "base_link", rosrust::Time::new())
-        .unwrap()
-        .transform;
+// 1mm and slightly less than half a degree
+fn poses_are_close(p1: &(f64, f64, f64), p2: &(f64, f64, f64)) -> bool {
```

Comment:
the name suggest poses as input but this accepts a tuple of three. Change to positions

Reply:
it's a 2D pose: x, y, yaw. But your confusion proves Carsten's point: we need a struct